tasks/databases.yml

```diff
@@ -5,4 +5,5 @@
     collation: "{{ item.collation | default('utf8_general_ci') }}"
     encoding: "{{ item.encoding | default('utf8') }}"
     state: present
+  sudo: no
```

Comment:
Agreed, it's kind of unwieldy, but still seems prudent to me.  In a given playbook probably most tasks don't really need sudo, but it seems like people tend to run them with `sudo: yes` just to make life easy.  I believe it's helpful to be explicit here so the role runs the same whether it's invoked with or without sudo.  I could go either way on that, let me know if you'd still like it changed.